crates/egui/src/data/input.rs

```diff
@@ -445,6 +445,12 @@ pub enum Event {
     /// * `zoom > 1`: pinch spread
     Zoom(f32),
 
+    /// IME Enable
+    ImeEnable,
+
+    /// IME Disable
+    ImeDisable,
+
     /// IME composition start.
     CompositionStart,
```

Comment:
Please add docstrings that explain these things. What does it mean to enable IME? How does that differ from `CompositionStart`?

Reply:
CompositionStart was created by egui,
ImeEnable is what happens by default in Windows (winit?).

in China's IME, characters are sometimes added without CompositionStart (Preedit?).

Reply:
I think, `Event::CompositionStart` may be `deprecated`.

Alternatively, if there is no confusion, we can use other names instead of winit's `Event::ImeEnable` and 'Event::ImeDisable' to 'Event::CompositionStart` and 'Event::CompositionEnd`.

Reply:
Please see if my comited code works well for you

Reply:
> Please see if my comited code works well for you

Yes, it works fine.

AND,
I would like to save the `TextEdit ID` in the same format as #4374 to prevent the entered text from being copied to another TextEdit.

* Related #4374